### PR TITLE
Translated unknown-prop page to PT-BR.

### PR DESCRIPTION
**Pull request by crisgon**

### CODE REVIEW

File: content/warnings/unknown-prop.md [hunk 1]
```diff
@@ -3,37 +3,41 @@ title: Unknown Prop Warning
 layout: single
 permalink: warnings/unknown-prop.html
 ---
-The unknown-prop warning will fire if you attempt to render a DOM element with a prop that is not recognized by React as a legal DOM attribute/property. You should ensure that your DOM elements do not have spurious props floating around.
+O aviso unknown-prop (propriedade desconhecida) será disparado se você tentar renderizar um elemento DOM com a prop que não é reconhecida pelo React como um atributo/propriedade DOM válido.  Você deve garantir que seus elementos DOM não tenham props falsas declaradas.
```
Comment (fjoshuajr) [inline content/warnings/unknown-prop.md]:
```suggestion
O aviso _unknown-prop_ (propriedade desconhecida) será disparado se você tentar renderizar um elemento DOM com a prop que não é reconhecida pelo React como um atributo/propriedade DOM válido. Você deve garantir que seus elementos DOM não tenham props falsas declaradas.
```

File: content/warnings/unknown-prop.md [hunk 2]
```diff
@@ -3,37 +3,41 @@ title: Unknown Prop Warning
...
+1. Você está usando `{...this.props}` ou `cloneElement(element, this.props)`? Seu componente está transferindo suas próprias props diretamente para um elemento filho (por exemplo. [transferindo props](/docs/transferring-props.html)). Quando transferir props para um elemento filho, você deve garantir que não está passando acidentalmente props que deveriam ser interpretadas pelo componente pai.
 
-4. You are using a React component without an upper case. React interprets it as a DOM tag because [React JSX transform uses the upper vs. lower case convention to distinguish between user-defined components and DOM tags](/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized).
+2. Você está usando um atributo DOM não padrão em um elemento DOM nativo, possivelmente para representar dados personalizados. Se você está tentando inserir um dado personalizado em um elemento padrão do DOM, considere usar um atributo data como descrito [em MDN](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes).
```
Comment (jhonmike) [inline content/warnings/unknown-prop.md]:
```suggestion
2. Você está usando um atributo DOM não padrão em um elemento DOM nativo, possivelmente para representar dados personalizados. Se você está tentando inserir um dado personalizado em um elemento padrão do DOM, considere usar um atributo data como descrito [em MDN](https://developer.mozilla.org/pt-BR/docs/Web/Guide/HTML/Using_data_attributes).
```
has an extra ENTER at the end of that phrase.

Comment (fjoshuajr) [inline content/warnings/unknown-prop.md]:
```suggestion
2. Você está usando um atributo DOM não padrão em um elemento DOM nativo, possivelmente para representar dados personalizados. Se você está tentando inserir um dado personalizado em um elemento padrão do DOM, considere usar um atributo data como descrito [em MDN](https://developer.mozilla.org/pt-BR/docs/Web/Guide/HTML/Using_data_attributes).
```

File: content/warnings/unknown-prop.md [hunk 3]
```diff
@@ -3,37 +3,41 @@ title: Unknown Prop Warning
...
+
+
+3. O React ainda não reconhece o atributo que você especificou. Isto provavelmente será corrigido em uma versão futura do React. Entretanto, o React atualmente retira todos os atributos desconhecidos, portanto especificá-los em seu aplicativo React não fará com que eles sejam renderizados.
+
+4. Você está usando um componente React sem uma letra maiúscula. O React interpreta isso como uma tag DOM, porque [O React JSX usa letras maiúsculas e minúsculas para distinguir entre componentes definidos pelo usuário e tags DOM](/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized).
```
Comment (fjoshuajr) [inline content/warnings/unknown-prop.md]:
```suggestion
4. Você está usando um componente React sem uma letra maiúscula. O React interpreta isso como uma tag DOM, porque [o React JSX usa letras maiúsculas e minúsculas para distinguir entre componentes definidos pelo usuário e tags DOM](/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized).
```

File: content/warnings/unknown-prop.md [hunk 4]
```diff
@@ -3,37 +3,41 @@ title: Unknown Prop Warning
 layout: single
 permalink: warnings/unknown-prop.html
 ---
-The unknown-prop warning will fire if you attempt to render a DOM element with a prop that is not recognized by React as a legal DOM attribute/property. You should ensure that your DOM elements do not have spurious props floating around.
+O aviso unknown-prop (propriedade desconhecida) será disparado se você tentar renderizar um elemento DOM com a prop que não é reconhecida pelo React como um atributo/propriedade DOM válido.  Você deve garantir que seus elementos DOM não tenham props falsas declaradas. 
 
-There are a couple of likely reasons this warning could be appearing:
+Existem algumas possíveis razões em que esses avisos podem aparecer:
```
Comment (fjoshuajr) [inline content/warnings/unknown-prop.md]:
```suggestion
Existem algumas possíveis razões desses avisos aparecerem:
```

**Comment by crisgon** [comment]
> Check the comments, please.

Done!

**Pull request merged**